### PR TITLE
Update virtual-merge

### PR DESCRIPTION
**Pull request by blalbrit**
add honeybadger requirement

### CODE REVIEW

**Comment by blalbrit** [comment]
Updated the other four scripts in /bin with the honeybadger requirement.

File: bin/decommission [hunk 1]
```diff
@@ -2,6 +2,8 @@
 USAGE = "Usage: decomission [-e ENVIRONMENT] druid:aa123bb4567|druids.txt \"Decomission tag\""
 require_relative '../config/boot'
 
+require 'honeybadger'
```
Comment (cbeer) [inline bin/decommission]: Can you put this in `config/boot` instead of duplicating it in every file?

Comment (tingulfsen) [inline bin/decommission]: ah - ok yeah that was my suggestion, config/boot is better

**Comment by blalbrit** [comment]
Okay - reversed individual bin/script changes and added to config/boot.

**Comment by tingulfsen** [comment]
Closing in favor of #19

**Pull request closed without merging**